### PR TITLE
Support Ubuntu 18.04

### PR DESCRIPTION
**Pull request by kmaehashi**

### CODE REVIEW

**Comment by kmaehashi** [comment]
Jenkins, test this please.

**Comment by kmaehashi** [comment]
Jenkins, test this please.

**Comment by kmaehashi** [comment]
Jenkins, test this please.

File: docker.py [hunk 1]
```diff
@@ -254,6 +256,11 @@ def get_numpy_choices():
     python_ver='.'.join(
         [str(x) for x in get_python_version('ubuntu16_py37-pyenv')]),
 )
+codes['ubuntu18_py37-pyenv'] = ubuntu_pyenv_base.format(
+    ubuntu_ver='18.04',
+    python_ver='.'.join(
+        [str(x) for x in get_python_version('ubuntu16_py37-pyenv')]),
```
Comment (toslunar) [inline docker.py]:
```suggestion
        [str(x) for x in get_python_version('ubuntu18_py37-pyenv')]),
```

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by toslunar** [comment]
Jenkins, test this please.

**Pull request merged**